crates/krun/src/bin/krun.rs

```diff
@@ -206,10 +207,32 @@ fn main() -> Result<()> {
         "PATH",                        // needed by `krun-guest` program
         "RUST_LOG",
     ];
+
+    // https://github.com/AsahiLinux/docs/wiki/Devices
+    const ASAHI_SOC_DEV_IDS: [&str; 12] = [
+        "t8103", "t6000", "t6001", "t6002", "t8112", "t6020", "t6021", "t6022", "t8122", "t6030",
+        "t6031", "t6034",
+    ];
     for key in WELL_KNOWN_ENV_VARS {
         let value = match env::var(key) {
             Ok(value) => value,
             Err(VarError::NotPresent) => {
+                if key == "MESA_LOADER_DRIVER_OVERRIDE" {
+                    match fs::read_to_string("/proc/device-tree/compatible") {
+                        Ok(compatible) => {
+                            // ...
+                        },
+                        Err(err) => if err.kind() == ErrorKind::NotFound {
```

Comment:
Oops. I had a typo here.
```suggestion
                        Err(err) if err.kind() == ErrorKind::NotFound => {
```